src/main/java/meditracker/exception/DuplicateArgumentFoundException.java

```diff
@@ -7,4 +7,7 @@ public class DuplicateArgumentFoundException extends Exception {
     public DuplicateArgumentFoundException(String errorContext) {
         super(errorContext);
     }
+
+    public DuplicateArgumentFoundException() {
```

Comment:
Is this empty constructor being used by the code?

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/218/commits/f600cce87c24fdfec92f3bd1c50836666b14f4fd